Rb.Forms.Barcode.nuspec

```diff
@@ -3,42 +3,33 @@
   <metadata>
     <id>Rb.Forms.Barcode</id>
     <title>Barcode scanner plugin for Xamarin.Forms</title>
-    <version>0.4.2-beta</version>
+    <version>0.5.0-beta</version>
     <authors>Ota Mares</authors>
     <owners>reBuy reCommerce GmbH</owners>
     <licenseUrl>https://github.com/rebuy-de/rb-forms-barcode/blob/master/LICENSE</licenseUrl>
     <projectUrl>https://github.com/rebuy-de/rb-forms-barcode</projectUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <description>Rb.Forms.Barcode is a Xamarin.Forms view for scanning barcodes. Build upon the decoding capabilities of ZXing.Net.Mobile, full of awesomeness and also unicorns.
-Please check the website for further details.
+    <description>Rb.Forms.Barcode is a Xamarin.Forms view for scanning barcodes full of awesomeness and also unicorns. Please check the website for further details.
 
 Beta notice: Currently only android is supported.</description>
     <summary>Rb.Forms.Barcode is a Xamarin.Forms view for scanning barcodes.</summary>
-    <releaseNotes>0.4.2-beta:
-* Fixed: NullReferenceException
-* Improved: Updated to Xamarin.Forms 1.4.4.6392
-* Improved: Updated to ZXing.Net.Mobile 1.5.1
-* Improved: Updated to Xamarin.Android.Support.v4 22.2.1
+    <releaseNotes>0.5.0-beta:
+Please note that version 0.5.0-beta contains api breaks and most of the automatic camera control logic was removed. Thus you should check if your app still works as desired. To fix arising issues you probably have to weave in camera control code into your app.
 
-0.4.1-beta:
-* Fixed: Handle TaskCanceledException
-* Improved: Updated to Xamarin.Forms 1.4.3
-
-0.4.0-beta:
-* New: Add every event as command
+* New: Use Google Play Services Vision API
+* New: Added torch control
+* Changed: Removed most of the automatic camera control logic as it adds undesired complexity to the library.
+* Changed: Dropped ZXing dependency
+* Improved: Updated Xamarin.Forms to version 1.5.0.6446
 </releaseNotes>
     <tags>barcode scanner scanning scan zxing xamarin xamarin.forms rebuy unicorns</tags>
     <dependencies>
         <group>
-            <dependency id="Xamarin.Forms" version="1.4.4.6392" />
-        </group>
-
-        <group targetFramework="MonoAndroid10">
-            <dependency id="ZXing.Net.Mobile" version="1.5.1.0" />
+            <dependency id="Xamarin.Forms" version="1.5.0.6446" />
         </group>
 
         <group targetFramework="MonoAndroid10">
-            <dependency id="Xamarin.Android.Support.v4" version="22.2.1.0" />
```

Comment:
Yes, Xamarin Google Play Services - Vision has a dependency on Xamarin Google Play Services - Base and that has a reference on the support package.